tests/django_restframework_gis_tests/models.py

```diff
@@ -43,15 +43,22 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
 
-class Location(BaseModel):
-    geometry = models.GeometryField(null=True, blank=True)
```

Comment:
Main mistakes here were two:
1) the `Location` model was later used as a base class, but we actually need another `abstract` model that has the geometry field (now named `BaseModelGeometry`).
2) the geometry field was set to be `null=True, blank=True`, while in the original code this was not the case. There is a separate `Nullable` class that already exists doing that.